README.md

```diff
@@ -33,6 +33,17 @@ It will not:
 * Cookbooks:
   * Sander van Zoest sysctl `https://github.com/svanzoest-cookbooks/sysctl`
 
+### Platform
+
+- Debian 7, 8
+- Ubuntu 14.04, 16.04
+- RHEL 6, 7
+- CentOS 6, 7
+- Oracle Linux 6, 7
+- Fedora 24, 25
```

Comment:
It should be 26 and 27

Reply:
Will fix that.

Reply:
I move this to the other PR to, so that this PR focus on amazon linux

Reply:
Great!